Network/NetworkTraceroute.cpp

```diff
@@ -101,21 +100,41 @@ namespace WPEFramework {
                 }
                 else
                 {
-                    snprintf(command, MAX_COMMAND_LENGTH, CMD_TRACEROUTE,
-                            wait,
-                            maxHops,
-                            packets,
-                            endpoint.c_str(),
-                            DEFAULT_IPV4_PACKET_LENGTH);
+                   pipe = v_secure_popen("r", "traceroute -w %d -m %d -q %d %s %d 2>&1",
+                           wait,
+                           maxHops,
+                           packets,
+                           endpoint.c_str(),
+                           DEFAULT_IPV4_PACKET_LENGTH);
                 }
 
-                if (NetUtils::execCmd(command, output, &result) < 0)
+                if (pipe == NULL)
```

Comment:
Do we need to remove NetUtils::execCmd? We can just change the popen call within this wrapper API to v_secure_popen so that other modules within Network can continue to use the helper function execCmd.

Reply:
@anand-ky, thanks for your comment.
v_secure_popen() does not accept the command argument as string. So 
pipe = popen(commandString.c_str(), "r"); 
can not be replaced as 
pipe = v_secure_popen("r", commandString.c_str());
Also the command must be a static string (with formatting), … cannot be a variable string.
I'm attaching and email from David,Jones explaining this :

![image](https://github.com/rdkcentral/rdkservices/assets/114486962/1ad9ad9f-ba57-46c2-9be3-943ed3059fad)

Reply:
sounds good, thanks.